### PR TITLE
Overhaul Infra docs - make Stewart a Super Admin

### PR DESCRIPTION
**Pull request by karianna**

### CODE REVIEW

File: CHAOS_MONKEY.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+and recreate it completely from our Ansible playbooks.
+
+## Security and Patching
+
+Ansible must ensure that the underlying O/S is patched as well as any firewalls, 
+VPN and other security configured before making the host available.
+
+## Core Infrastructure
+
+Provider | Host | role | Chaos Monkey? | Issue(s) |
```
Comment (sxa555) [inline CHAOS_MONKEY.md]: I presume "issue" here is intended to be a link to an issue that would cover fixing any problem with the ansible scripts not regenerating everything as expected?

File: README.md [hunk 2]
```diff
@@ -1,27 +1,86 @@
...
 # Members
 
-#### [@admin_infrastructure](https://github.com/orgs/AdoptOpenJDK/teams/admin_infrastructure)
+We list members and their organisation affiliation for maximum transparency. Want to add 
+a new member? Please follow our [Onboarding Process](ONBOARDING.md). 
 
-- [@gdams](https://github.com/gdams) - George Adams (IBM)
-- [@johnoliver](https://github.com/johnoliver) - John Oliver (jClarity / LJC)
-- [@mwornast](https://github.com/mwornast) - Marcus Wornast (IBM)
+## [@admin_infrastructure](https://github.com/orgs/AdoptOpenJDK/teams/admin_infrastructure)
```
Comment (sxa555) [inline README.md]: Maybe I've missed it but what's the `*` against someone's name intended to indicate?

File: README.md [hunk 3]
```diff
@@ -74,20 +137,21 @@ Backup retention:
 - How long should be keep it?
 - How many copies?
 
-#### OS Patch Management: 
-*Most of our systems are internet facing and we need to stay vigilant of the potential security risks this presents.*
+## OS Patch Management
+**WARN:** Several of our hosts are internet facing and we need to stay vigilant
```
Comment (sxa555) [inline README.md]: Should this be "WARNING"?

File: CHAOS_MONKEY.md (hunk 1 above)
Comment (karianna) [reply CHAOS_MONKEY.md]: Yes

File: README.md (hunk 2 above)
Comment (karianna) [reply README.md]: Access to the secrets repo.  I need to document that.

**Pull request merged**